apps/chat-web/app/components/langchain-chat-form.tsx

```diff
@@ -37,15 +37,15 @@ export const LangchainChatForm = ({ sessionId }: { sessionId: string }) => {
               sender: 'user',
               text: message,
               source: '',
-              time: new Date(),
+              time: new Date(Date.now()),
```

Comment:
```suggestion
              time: new Date(),
```
[Look at the doc for Date()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date)